crates/agent-sql/src/connector_tags.rs

```diff
@@ -42,6 +42,47 @@ pub async fn dequeue(txn: &mut sqlx::Transaction<'_, sqlx::Postgres>) -> sqlx::R
     .await
 }
 
+/// JobStatus is the possible outcomes of a handled connector tag.
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase", tag = "type")]
+pub enum JobStatus {
```

Comment:
The intent of the separation of these crates is that agent-sql is _only_ queries and structs which are POD of query outcomes.

I see why you moved it, but it's a smell that you had too.

Reply:
Moved it back